.github/workflows/chainloop.yml

```diff
@@ -1,10 +1,13 @@
-name: Chainloop Metadata Collection and Attestation Process
+name: Chainloop Attestation Process
 on:
   workflow_call:
     inputs:
       chainloop_version:
         required: false
         type: string
+      contract_revision:
+        required: false
+        type: string
     secrets:
```

Comment:
for private repository, do you need to configure read permissions on the repo so the retrieve logs work?

Reply:
I was testing it with the private repo and did not have to add anything